.travis.yml

```diff
@@ -1,6 +1,3 @@
-# Tell Travis what to do
-# https://travis-ci.com/nedbat/coveragepy
-
```

Comment:
Why remove the comment from the top?

Reply:
I found it not useful, after having read it a few times - but that's because I know what it is for.  I can leave it in.